src/libraries/System.Private.Xml/src/System/Xml/Xsl/Runtime/NumberFormatter.cs

```diff
@@ -93,7 +93,7 @@ public static void ConvertToAlphabetic(StringBuilder sb, double val, char firstC
                 number = quot;
             }
             letters[--idx] = (char)(firstChar + --number);
-            sb.Append(letters, idx, MaxAlphabeticLength - idx);
+            sb.Append(letters.Slice(idx, MaxAlphabeticLength - idx));
```

Comment:
Is the `, MaxAlphabeticLength - idx` part needed? It couldn't just be `sb.Append(letters.Slice(id))`?

Reply:
I think it is the same , just wanted to keep the code almost the same as before.